src/pages/Validator/index.tsx

```diff
@@ -4,17 +4,31 @@ import {useParams} from "react-router-dom";
 import PageHeader from "../layout/PageHeader";
 import ValidatorTitle from "./Title";
 import ValidatorDetailCard from "./DetailCard";
+import ValidatorStakingBar from "./StakingBar";
+import {useGetMainnetValidators} from "../../api/hooks/useGetValidatorSet";
+import EmptyValue from "../../components/IndividualPageContent/ContentValue/EmptyValue";
 
 export default function ValidatorPage() {
   const address = useParams().address ?? "";
+  // make sure that addresses will always start with "0x"
+  const addressHex = address.startsWith("0x") ? address : "0x" + address;
```

Comment:
should this be a global util func? I think I've seen it somewhere else also. another option is to utilize the ts sdk and do `const addressHex = new HexString(address);`
https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/hex_string.ts#L65